LayoutTests/platform/ios/TestExpectations

```diff
@@ -8219,6 +8219,9 @@ fast/forms/ios/form-control-refresh/color/paint-within-box.html [ ImageOnlyFailu
 fast/forms/ios/form-control-refresh/select/decoration-color.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/fractional-translation-from-position.html [ ImageOnlyFailure ]
 
+# <rdar://156470462> RTL inputs with type=password have larger logical width than RTL inputs with type number, url, tel, & text
+fast/forms/input-logical-widths.html [ Failure ]
```

Comment:
It's not great that we need to mark the entire test as a failure because of one input type? Can the test be split up?